libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -171,36 +169,37 @@
     (click)="onInsetIconClick()"
   >
     <ng-content select=".sky-input-group-icon.sky-input-box-icon-inset-left" />
-    <ng-container
-      *ngIf="hostIconsInsetLeftTemplate"
-      [ngTemplateOutlet]="hostIconsInsetLeftTemplate"
-    />
+    @if (hostIconsInsetLeftTemplate) {
+      <ng-container [ngTemplateOutlet]="hostIconsInsetLeftTemplate" />
+    }
   </div>
 </ng-template>
 
 <ng-template #errorLabelTemplate>
```

Comment:
the below chunk of code adding `sky-input-box-errors` is the only relevant change in this file.